crates/common_enums/src/enums.rs

```diff
@@ -1643,6 +1643,33 @@ pub enum PaymentMethod {
     MobilePayment,
 }
 
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Default,
+    Eq,
+    Hash,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    strum::Display,
+    strum::VariantNames,
+    strum::EnumIter,
+    strum::EnumString,
+    ToSchema,
+)]
+#[strum(serialize_all = "snake_case")]
+pub enum PaymentMethodStage {
+    /// Payment Method available in production
+    #[default]
```

Comment:
remove the default value, we should not infer any default values

Reply:
This field is removed